package.json

```diff
@@ -50,7 +50,7 @@
     "babel-plugin-module-resolver": "^3.1.1",
     "babel-runtime": "^6.26.0",
     "buffer": "^5.1.0",
-    "did-resolver": "0.0.6",
+    "did-resolver": "uport-project/did-resolver#feature/unglobalize",
```

Comment:
we should not be merging dependencies that point to git branches

Reply:
yep I'm keeping the [WIP] label until did-resolver is released and we can use the new version of that in here

Reply:
did-resolver 1.0.0 has been released